app/routes.js

```diff
@@ -49,4 +50,6 @@ exports.init = app => {
     ],
     userController.userNewSession
   );
+
+  app.get('/users', [jwt({ secret: config.secret })], userController.usersList);
```

Comment:
I would personally prefer to move this to another middleware file and abstract which package or logic I use there and here only call that customizable middleware.
But this is correct.

Reply:
Done that in #13